src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -163,6 +163,26 @@ public void ToggleEngulf()
         }
     }
 
+    [RunOnKeyDown("g_eject_engulfed")]
+    public void EjectAllEngulfed()
+    {
+        ref var engulfer = ref stage.Player.Get<Engulfer>();
+
+        if (engulfer.EngulfedObjects == null || engulfer.EngulfedObjects?.Count == 0)
```

Comment:
Same shorter check should be usable here as well than what I commented on before.